test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGet.PackageManagement.Test.csproj

```diff
@@ -3,6 +3,8 @@
     <TargetFrameworks>$(TargetFrameworksUnitTest)</TargetFrameworks>
     <UseParallelXunit>true</UseParallelXunit>
     <Description>Unit tests for NuGet.PackageManagement.</Description>
+    <TestProjectType Condition="'$(IsXPlat)' != 'true'">Functional</TestProjectType>
```

Comment:
This is how a project conveys that its a functional test but only on Windows.

Reply:
Yes, in this case the `Directory.Build.props` actually indicates that every project in this folder is a unit test.  But this test project is actually a Functional test.  Its currently declared like this:

https://github.com/NuGet/NuGet.Client/blob/dev/build/common.project.props#L210-L216